<!-- -*- mode: XML -*- -->

<launch>
    <node name="dynamixel_manager" pkg="dynamixel_controllers" type="controller_manager.py" required="true" output="screen">
        <rosparam>
            namespace: dxl_manager
            serial_ports:
                arm_port:
                    port_name: "/dev/ttyACM0"
                    baud_rate: 1000000
                    min_motor_id: 1
                    max_motor_id: 25
                    update_rate: 20
        </rosparam>
    </node>


    <!-- Start tilt joint controller -->
    <rosparam file="$(find eurobot2014_gros_arm)/launch/right_arm.yaml" command="load"/>
    <node name="right_arm_controller_spawner" pkg="dynamixel_controllers" type="controller_spawner.py"
          args="--manager=dxl_manager
                --port arm_port
                Rlink1_controller
                Rlink2_controller
                Rlink3_controller
                Rlink4_controller
                Rlink5_controller"
          output="screen"/>

</launch>
